doc/transport/input.rst

```diff
@@ -0,0 +1,240 @@
+----------------------------------------------
+
+Measure
+   Passenger distance travelled per capita in the model base year
+Dimensions
+   :math:`(n)`
+Units
+   km / year
+
+Notes
+~~~~~
+
+node = R12_CHN [4]_
+   Based on the vehicle activity method `Liu, et al. 2022`_ estimate the total PDT for R12_CHN for year (2017) is 9406 billion pkm.
+   This is the latest corrected estimate available from Liu, et al. 2022.
+   Based on similar estimates for 2013 & 2015, I estimate the average growth of PDT to be 8% per year.
+   Using the growth rate and 2017 estimate, the total PDT for year (2020) comes out to be 11848.9 billion pkm.
+
+   R12_CHN population estimate from IMAGE: 1.4483 billion
+
+   the PDT/capita = 11848.9/1.4483
+
+.. [4] A. Javaid, `message_data#538 (comment) <https://github.com/iiasa/message_data/issues/538#issuecomment-1934663340>`__.
+
+:file:`pdt-elasticity.csv` → ``pdt elasticity:scenario-n:exo``
+--------------------------------------------------------------
+
+Measure
+   “Elasticity” or multiplier for GDP PPP per capita
+Dimensions
+   :math:`(n, \text{scenario})`.
+   ‘scenario’ identifiers are partial URNs for codes in the :class:`.SSP_2024` code list.
+Units
+   dimensionless
+Where/how used
+   :func:`.pdt_per_capita`.
+
+Other files
+-----------
+- :file:`demand-scale.csv` → ``demand scale:n-y:exo``
+- :file:`disutility.csv` → ``disutility:n-cg-t-y:per vehicle``
+- :file:`energy-other.csv` → ``energy:c-n:transport other``
+- :file:`freight-activity.csv` → ``freight activity:n:ref``
+- :file:`freight-mode-share-ref.csv` → ``freight mode share:n-t:ref``
+- :file:`fuel-emi-intensity.csv` → ``fuel emi intensity:c-e:exo``
+- :file:`ikarus/availability.csv` → ``ikarus availability:source-t-c-y:exo``
+- :file:`ikarus/fix_cost.csv` → ``ikarus fix_cost:source-t-c-y:exo``
+- :file:`ikarus/input.csv` → ``ikarus input:source-t-c-y:exo``
+- :file:`ikarus/inv_cost.csv` → ``ikarus inv_cost:source-t-c-y:exo``
+- :file:`ikarus/technical_lifetime.csv` → ``ikarus technical_lifetime:source-t-c-y:exo``
+- :file:`ikarus/var_cost.csv` → ``ikarus var_cost:source-t-c-y:exo``
+- :file:`input-base.csv` → ``input:t-c-h:base``
+- :file:`ldv-class.csv` → ``ldv class:n-vehicle_class:exo``
+- :file:`ldv-new-capacity.csv` → ``cap_new:nl-t-yv:ldv+exo``
+- :file:`load-factor-ldv.csv` → ``load factor ldv:n:exo``
+- :file:`load-factor-nonldv.csv` → ``load factor nonldv:t:exo``
+- :file:`ma3t/attitude.csv` → ``ma3t attitude:attitude:exo``
+- :file:`ma3t/driver.csv` → ``ma3t driver:census_division-area_type-driver_type:exo``
+- :file:`ma3t/population.csv` → ``ma3t population:census_division-area_type:exo``
+- :file:`mer-to-ppp.csv` → ``mer to ppp:n-y:exo``
+- :file:`population-suburb-share.csv` → ``population suburb share:n-y:exo``
+
+Other data sources
+==================
+
+:mod:`~message_data.model.transport` makes use of the :mod:`message_ix_models.tools.exo_data` mechanism to retrieve data from common (not transport-specific) sources.
```

Comment:
Is this a forgotten mention of message_data?

Reply:
Yes, will update via #210.